leadbutt.py

```diff
@@ -133,16 +134,31 @@ def leadbutt(config_file, cli_options, verbose=False, **kwargs):
         end_time = datetime.datetime.utcnow()
         start_time = end_time - datetime.timedelta(
             seconds=period_local * count_local)
-        results = conn.get_metric_statistics(
-            period_local,  # minimum: 60
-            start_time,
-            end_time,
-            metric['MetricName'],  # RequestCount, CPUUtilization
-            metric['Namespace'],  # AWS/ELB, AWS/EC2
-            metric['Statistics'],  # Sum, Maximum
-            dimensions=metric['Dimensions'],
-            unit=metric['Unit'],  # Count, Percent
-        )
+
+        # if the metric is something that AWS only give you once per day
+        if metric['Namespace'] == "AWS/S3" or metric['Namespace'] == "AWS/Redshift":
```

Comment:
An array of 'once a day' metrics would be preferable; in `__init__.py` would be ideal, but we don't have that; in any case, loadable from the other script/module if possible?